<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>824</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Begin OpenGL</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetDefaultConstraint</enum>
      </property>
      <property name="horizontalSpacing">
       <number>2</number>
      </property>
      <property name="verticalSpacing">
       <number>1</number>
      </property>
      <item row="7" column="0">
       <widget class="QLabel" name="translate_z_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Translate Z:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="translate_x_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Translate X:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2" alignment="Qt::AlignVCenter">
       <widget class="QSlider" name="x_rot_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font/>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #FF0000;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="7" column="2">
       <widget class="QSlider" name="z_trans_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #00FF00;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0" colspan="3">
       <widget class="MyGL" name="openGLWidget">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="rotate_z_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Rotate Z:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="rotate_x_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Rotate X:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="translate_y_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Translate Y:</string>
        </property>
       </widget>
      </item>
      <item row="4" column="2">
       <widget class="QSlider" name="z_rot_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #FF0000;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="2">
       <widget class="QSlider" name="y_trans_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #00FF00;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="rotate_y_label">
        <property name="font">
         <font>
          <italic>true</italic>
          <bold>true</bold>
          <underline>false</underline>
          <strikeout>false</strikeout>
         </font>
        </property>
        <property name="text">
         <string>Rotate Y:</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2">
       <widget class="QSlider" name="x_trans_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #00FF00;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="sliderPosition">
         <number>0</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="2">
       <widget class="QSlider" name="y_rot_slider">
        <property name="minimumSize">
         <size>
          <width>703</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
    border: 1px solid FF0000;
    height: 8px; 
   /* background: #D3D3D3;*/
 background: qlineargradient(x2:0, y1:0, x2:1, y2:1, stop:0 #B1B1B1, stop:1 #c4c4c4);
    margin: 2px 0;
}

QSlider::handle:horizontal {
    background: qlineargradient(x1:0, y1:0, x2:1, y2:1, stop:0 #b2b2b2, stop:1 #6f6f6f);
    border: 1px solid #FF0000;
    width: 18px;
    margin: -2px 0; 
    border-radius: 3px;
}</string>
        </property>
        <property name="minimum">
         <number>0</number>
        </property>
        <property name="maximum">
         <number>360</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="8" column="2">
       <widget class="QPushButton" name="quitButton">
        <property name="minimumSize">
         <size>
          <width>717</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <bold>true</bold>
         </font>
        </property>
        <property name="autoFillBackground">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton {
  background-color: rgb(215, 215, 215);
  border: 1px solid gray;
 color : black;
	border-radius: 4px;
}
</string>
        </property>
        <property name="text">
         <string>QUIT</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyGL</class>
   <extends>QOpenGLWidget</extends>
   <header>my_gl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
